source/manuals/accessibility.html.md.erb

```diff
@@ -36,11 +36,13 @@ This is why [semantic markup](https://html.com/semantic-markup/) and good headin
 
 Accessibility testing tools such as [aXe](https://www.deque.com/axe/) and [Lighthouse](https://developers.google.com/web/tools/lighthouse/) are useful for finding issues, but [automated testing can only find around 30% of likely accessibility problems](https://alphagov.github.io/accessibility-tool-audit/). Automated testing shouldn’t be the only accessibility testing carried out on a service, it should also be tested with real users.
 
-### Browser compatibility testing
+### Testing with assistive technologies
 
-Services should be thoroughly tested in all browsers they are required to be compatible with, including mobile devices and accessibility tools such as screen readers. Emulated browser testing services are available, but some testing on real mobile devices is also recommended.
+Test your service with assistive technologies throughout development, especially when you introduce a significant feature or make any major change. You should test in [the assistive technology and browser combinations listed in the service manual](https://www.gov.uk/service-manual/technology/testing-with-assistive-technologies).
 
-For reference, these are the [browsers that GOV.UK is required to be compatible with](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices) and a service should also work with these [assistive technologies](https://www.gov.uk/service-manual/technology/testing-with-assistive-technologies#what-to-test).
+You can use [Assistiv Labs](https://assistivlabs.com/) to test your service with JAWS and NVDA. Contact Matt Hobbs (Head of Frontend) to request an account.
```

Comment:
Could we expand the acronyms on first use. I got a picture of a shark when I googled for JAWS :wink:

Reply:
Happy to, but not sure that it helps the user much… we don't expand them in the service manual either?